azure_monitor/src/azure_monitor/sdk/auto_collection/metrics_span_processor.py

```diff
@@ -13,49 +12,21 @@
 
 class AzureMetricsSpanProcessor(SpanProcessor):
```

Comment:
We still need this class to collect live metrics documents, even if they are sampled

Reply:
Can you move this under `live_metrics` folder?

Reply:
I believe this is still under `auto_collection`